src/main/java/org/jetbrains/bsp/bazel/server/bazel/data/SemanticVersion.java

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.bsp.bazel.server.bazel.data;
+
+public class SemanticVersion implements Comparable<SemanticVersion> {
+
+  private final String version;
+
+  public final String get() {
+    return this.version;
+  }
+
+  public SemanticVersion(String version) {
+    if (version == null) throw new IllegalArgumentException("Version can not be null");
+    if (!version.matches("[0-9]+(\\.[0-9]+)*"))
```

Comment:
I believe that it would be better to extract this regex to the variable with proper name

Reply:
and again brackets